testing/_index.md

```diff
@@ -9,7 +9,17 @@ chapter: true
 
 # Testing
 
-This section describes how PrestaShop Core is covered by automatic tests.
+# How testing works in PrestaShop
```

Comment:
```suggestion
## How testing works in PrestaShop
```
There is already a heading maybe it's a heading 2 ?

Reply:
Oops